src/player_display.cpp

```diff
@@ -674,11 +679,34 @@ static void draw_skills_tab( const catacurses::window &w_skills, const catacurse
     }
 
     const Skill *selectedSkill = nullptr;
-
-    for( size_t i = min; i < max; i++ ) {
+    Skill::skill_type prev_type = Skill::skill_num_entries;
+    int y_pos = 1;
+    for( size_t i = min; i < max; i++, y_pos++ ) {
         const Skill *aSkill = skillslist[i];
         const SkillLevel &level = you.get_skill_level_object( aSkill->ident() );
 
+        if( prev_type != aSkill->get_skill_type() ) {
+            prev_type = aSkill->get_skill_type();
+            std::string type_name;
+            switch( prev_type ) {
+            // *INDENT-OFF*
+            case Skill::crafting_skill:     type_name = "Crafting skills";  break;
```

Comment:
```suggestion
            case Skill::crafting_skill:     type_name = _( "Crafting skills" );  break;
```
The same for other categories below.